src/components/external_docs/NIHDataUseCertificationAgreement.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type NIHDataUseCertificationAgreementProps = {
+    className: string | null
+    showDownloadIcon: boolean | null
+}
+export const NIHDataUseCertificationAgreement = (props: NIHDataUseCertificationAgreementProps) => {
+    const {className, showDownloadIcon} = props;
+    return (
+        <a
+        href={'https://sharing.nih.gov/accessing-data/accessing-genomic-data/using-genomic-data-responsibly/nih-data-use-certification-agreement'}
+        target={'_blank'}
+        rel={'noreferrer'}
+        className={className ? className : ''}
```

Comment:
Could be a little simpler
```suggestion
        className={className ?? ''}
```